tests/test_app.py

```diff
@@ -35,14 +38,21 @@ def now():
 
 @responses.activate
 def test_token(client, now, enrollment, access_key):
-    responses.add(
+
+    def request_callback(request):
```

Comment:
See last lines in example of https://github.com/getsentry/responses#response-body-as-string